addons/account/models/account_payment.py

```diff
@@ -703,6 +705,7 @@ class payment_register(models.TransientModel):
                                         "Batch Deposit: Encase several customer checks at once by generating a batch deposit to submit to your bank. When encoding the bank statement in Odoo, you are suggested to reconcile the transaction with the batch deposit.To enable batch deposit, module account_batch_payment must be installed.\n"
                                         "SEPA Credit Transfer: Pay bill from a SEPA Credit Transfer file you submit to your bank. To enable sepa credit transfer, module account_sepa must be installed ")
     invoice_ids = fields.Many2many('account.invoice', 'account_invoice_payment_rel_transient', 'payment_id', 'invoice_id', string="Invoices", copy=False, readonly=True)
+    group_payment = fields.Boolean(help="Only one payment will be created by partner (bank)/ currency.")
```

Comment:
What the heck? 
Since when did we change the database structure in a stable version?

Reply:
I know, I know... As decided by AVW/AL, it had to be done in saas-12.3.